node/src/actors/chain_manager/mining.rs

```diff
@@ -916,37 +913,31 @@ fn build_block(
     transaction_fees += reveals_fees;
 
     // Include Mint Transaction by miner
-    let reward = block_reward(epoch) + transaction_fees;
+    let mut reward = block_reward(epoch) + transaction_fees;
 
     // Build Mint Transaction
-    let mint = if split_mint {
-        // The rewards will be split into multiple outputs of the same value as `collateral_minimum`
-        // until the node's set of outputs available for collateralization outnumbers
-        // `data_request_max_retrievals_per_epoch` multiplied by `collateral_coinage`,
-        // i.e. makes sure it always has spare outputs to collateralize.
-
-        let utxos_to_achieve = collateral_age
-            .saturating_mul(u32::from(data_request_max_retrievals_per_epoch))
-            as usize;
-        let utxos_required = utxos_to_achieve.saturating_sub(own_utxos.bigger_than_min_counter());
-
-        MintTransaction::with_split_utxos(
-            epoch,
-            reward,
-            own_pkh,
-            collateral_minimum,
-            utxos_required,
-        )
-    } else {
-        MintTransaction::new(
-            epoch,
-            vec![ValueTransferOutput {
-                pkh: own_pkh,
-                value: reward,
+    let mut vts = vec![];
+    if let Some(external_pkh) = external_address {
+        // In case of a specified pkh, the reward will be redistributed between the node's pkh
+        // and the specified one using the external_percentage value
+        let external_reward = (reward * u64::from(external_percentage)) / 100;
+        if external_reward > 0 {
+            reward -= external_reward;
+            vts.push(ValueTransferOutput {
+                pkh: external_pkh,
+                value: external_reward,
                 time_lock: 0,
-            }],
-        )
-    };
+            })
+        }
+    }
```

Comment:
This is a sort of weird positioning for this block, because unless I'm misreading, this will cause the classical self-address mint output to come second instead of first.

Reply:
yes, it was for simplify the conditions, but could be changed